docs/User-Experience/Research-Syntheses/2022, Fall - TDP 2.2 Pilot Expansion.md

```diff
@@ -0,0 +1,150 @@
+* **Observe** and document accessibility, browser and general IT compatibility concerns/issues
+* **Allocate** and conduct high level research to help inform decisions for future iterations 
+* **Build** a rapport with grantees for future expansion and utilization of the TANF Data Portal
+
+**Supporting Documentation**
+* [Pilot Workshop v2.2 :lock:](https://app.mural.co/t/raft2792/m/raft2792/1664205610518/2b3edb3a6f413ac420cffdc2461c9d5ecd1433e9?sender=u3ca60168b5ac4eb17bf92804)
+
+___
+
+### What we tested
+
+Each participant was tasked with the following:
+
+* **opening** TDP in a compatible browser (i.e. not choosing Internet Explorer)
+* **viewing** the home page
+* **signing-in** using Login.gov
+* **requesting** access to the TANF Data Portal
+* **receiving** inline email notifications from TANF Data Portal
+* **uploading** data files for FYQ4 2022 (July - September)
+* **submitting** data files for FYQ4 2022 (July - September)
+* **viewing** the submission notification (or submission error)
+
+Overall, the pilot expansion program was a success in which data was successfully transmitted for each grantee that participated. In addition, TDP has been utilized to help solve for instances where data file transfers have been a challenge due to existing file transfer protocols. 
+
+___
+
+### What we learned
+
+Jump to: 
+* [What went well](#What-went-well)
+* [Retest observed gaps from August 2022 pilot program](#Retest-observed-gaps-from-August-2022-pilot-program) 
+* [Some participants overlooked the Search functionality which may result in incorrectly categorized data file submissions](#Some-participants-overlooked-the-Search-functionality-which-may-result-in-incorrectly-categorized-data-file-submissions)
+* [Observed minor bugs during moderated sessions](#Observed-minor-bugs-during-moderated-sessions)
+* [Some participants expressed interest in being able to upload previous years data files](#Some-participants-expressed-interest-in-being-able-to-upload-previous-years-data-files)
+
+___
+
+### What went well
+
+Based on the task scenarios defined for v2.2 the pilot expansion program was a success. Grantees that participated were able to complete all task scenarios measured in v2.0 in addition to new released features for inline email notifications and being able to submit TANF and/or SSP-MOE data files. In addition, for those in our moderated sessions we were able to observe and record the efficiencies and perceived shortcomings of the current platform to help inform future iterations. While our top level findings focus on observed gaps to improve TDP, we continue to receive positive feedback on TDP’s impact from participating grantees.
+
+### Retest observed gaps from August 2022 pilot program 
+
+In addition to reviewing new functionality, we wanted to revisit and retest the areas noted from the August 2022 pilot program. Please reference [August 2022](https://tdp-project-updates.app.cloud.gov/index.html) for full details 
```

Comment:
@lilybl1 Should this be linked to the 2.0 synthesis rather than the TDP homepage?